Sources/PrimerSDK/Classes/User Interface/ApplePayPresentationManager.swift

```diff
@@ -33,13 +34,6 @@ class ApplePayPresentationManager: ApplePayPresenting, LogReporter {
     func present(withRequest applePayRequest: ApplePayRequest,
                  delegate: PKPaymentAuthorizationControllerDelegate) -> Promise<Void> {
         Promise { seal in
-            guard isPresentable else {
-                let error = errorForDisplay
-                ErrorHandler.handle(error: error)
-                seal.reject(error)
-                return
-            }
-
             let request = createRequest(for: applePayRequest)
 
             let paymentController = PKPaymentAuthorizationController(paymentRequest: request)
```

Comment:
Should we this opportunity to return a different error below when `success == false`
This way we could more easily see if we are failing in the isPresentable check, or when we actually try to present.

Reply:
We do add extra info in the `errorForDisplay` - so we should be able to tell the difference. But afaict that (along with the line number) is not making it to Elastic 🤔

Reply:
The error that triggers on `success == false` is a slightly different error: `unableToPresentApplePay` rather than `unableToPresentPaymentMethod`.